azafea/event_processors/endless/metrics/tests_v3/integration/test_metrics_v3.py

```diff
@@ -187,7 +187,8 @@ def test_singular_events(self):
             LaunchedExistingFlatpak, LaunchedInstallerForFlatpak, LinuxPackageOpened,
             ParentalControlsBlockedFlatpakInstall, ParentalControlsBlockedFlatpakRun,
             ProgramDumpedCore, UpdaterFailure, ParentalControlsEnabled,
-            ParentalControlsChanged, WindowsAppOpened, ComputerInformation
+            ParentalControlsChanged, WindowsAppOpened, ComputerInformation,
+            SplitFlatpakRepoStats,
```

Comment:
Thanks for adding the trailing comma. I think that in a different life it would be nice for these lists to be one-per-line and sorted alphabetically.

Reply:
Indeed, for another day.